s3prl_vc/models/diffsinger/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .denoiser import DiffNet
```

Comment:
I wonder if this file is really needed?

Reply:
This one is needed since we call these two classes inside the diffusion.py file.